### PR TITLE
Nerfs cats

### PR DESCRIPTION
**Pull request by listerla**
Cats can no longer spam your chat with seeing observers every five seconds. Or ever.

### CODE REVIEW

**Comment by listerla** [comment]
!review

**Comment by FayeLovesU** [comment]
no

**Comment by desvenlafaxine** [comment]
Honestly I don't see why, but if it's such an issue just lower the probability. I always thought it was a cute little thing that made the game more fun.

**Comment by listerla** [comment]
observers are not an IC thing and any mechanic that references them is annoying and bad
and 3/4th of the time the message is "near the firelock" or some shit

**Comment by listerla** [comment]
no changes will be made to this PR ever

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 1]
```diff
@@ -76,18 +76,6 @@
...
-			var/mob/abstract/observer/spook = locate() in range(src,5)
-			if(spook)
-				var/turf/T = spook.loc
-				var/list/visible = list()
-				for(var/obj/O in T.contents)
-					if(!O.invisibility && O.name)
-						visible += O
-				if(visible.len)
-					var/atom/A = pick(visible)
-					visible_emote("suddenly stops and stares at something unseen[istype(A) ? " near [A]":""].",0)
```
Comment (123khadi) [inline code/modules/mob/living/simple_animal/friendly/cat.dm:89]: so diffuclt cod e

**Review by 123khadi — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by 123khadi — COMMENTED** [review]
Yes are code is probably ok

**Pull request merged**